src/main/kotlin/dslstudy/Skill.kt

```diff
@@ -0,0 +1,7 @@
+package dslstudy
+
+sealed class Skill(open val name: String)
```

Comment:
name이 open으로 선언되어야할 이유가 있을까요?

```suggestion
sealed class Skill

data class Soft(val name: String) : Skill()

data class Hard(val name: String) : Skill()

```